Translations/ko.lproj/Credits.strings

```diff
@@ -1,15 +1,15 @@
-"_PROJECTTITLE_" = "The Keka project";
-"_PROGRAMMER_" = "Programmer, designer &amp; maintainer";
-"_ARTWORK_" = "Artwork designer";
-"_TRANSLATIONS_" = "Translations";
-"_TRANSLATIONSOLD_" = "Translations (from 0.1 to 0.1.4.3)";
-"_TESTERS_" = "Occasional testers";
-"_DONATIONS_" = "Donations (most sorted by date)";
+"_PROJECTTITLE_" = "Keka 프로젝트";
+"_PROGRAMMER_" = "프로그래머, 디자이너 &amp; 관리자(maintainer)";
```

Comment:
Previously, I was not sure that 관리자 can fully deliver the original meaning of "maintainer". Because 관리자 means the only administrator. So, I will change to "유지보수 및 관리 담당자" which means "people in charge of continuously fixing and administrating".